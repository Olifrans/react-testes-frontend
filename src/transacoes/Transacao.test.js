import React from "react";
import { render } from "@testing-library/react";
import Transacao from "./Transacao";

//Teste com Snapshot
describe("Componente de transação do extrato", () => {
  it("O Snapshot do componente deve permanecer sempre o mesmo", () => {
    const { container } = render(
      <Transacao data="23/02/2022" tipo="saque" valor="150.00" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});

