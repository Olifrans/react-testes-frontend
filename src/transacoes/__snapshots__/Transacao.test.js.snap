// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato O Snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    23/02/2022
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    150.00
  </p>
</div>
`;
